folium/features.py

```diff
@@ -643,6 +643,10 @@ class GeoJson(Layer):
                 .done({{ this.get_name() }}_add);
         {%- endif %}
 
+        {%- if not this.style %}
```

Comment:
I checked all the docs/notebooks we have and none broke. If the user specifies a `style_function`, whatever is in the geojson will be ignored but, if nothing is specified and the geojson has style info, that will be used instead. I guess this is the best I could come up with but I'm open to suggestions.

Reply:
Is this change required to fix this bug? It seems like a new feature. If so perhaps it is better to separate this into a new PR.

Also, what happens if a Feature in the GeoJson does not have a `style` property? Will leaflet handle this correctly?